docs/source/user_guide/pipelines.md

```diff
@@ -183,4 +176,35 @@ To export a pipeline from the Visual Pipeline Editor:
    
    ![Configure pipeline export options](../images/user_guide/pipelines/configure-pipeline-export-options.png)
 
-1. Import the exported pipeline file using the Kubeflow Central Dashboard or add it to the Git repository that Apache Airflow is monitoring.   
+1. Import the exported pipeline file using the Kubeflow Central Dashboard or add it to the Git repository that Apache Airflow is monitoring.
+
+
+#### Exporting a pipeline from the command line interface
+
+Use the [`elyra-pipeline`](command-line-interface.html#working-with-pipelines) `export` command to export a pipeline to a runtime-specific format, such as YAML for Kubeflow Pipelines or Python DAG for Apache Airflow.
+
+```bash
+$ elyra-pipeline export a-notebook.pipeline --runtime-config kfp_dev_env --output /path/to/exported.yaml --overwrite
```

Comment:
do we want to add a blurb here about how to go about getting the `--runtime-config`? 
e.g. try `elyra-metadata list runtimes` and find look under the `Instance` column for name. The instance selected must be compatible with the pipeline being exported. etc. etc.

Reply:
Done. Every command in this topic that requires `runtime-config` as input should now include a link to the relevant topic.